kanidmd/src/lib/constants/mod.rs

```diff
@@ -43,3 +43,6 @@ pub const PW_MIN_LENGTH: usize = 10;
 
 // Default
 pub const AUTH_SESSION_EXPIRY: u64 = 3600;
+
+/// Default max request post size, 100MB
+pub const MAX_BODY_SIZE: usize = 104857600;
```

Comment:
I probably unaware why we need a large request size.
100MB seems quit high for this application.
1MB or even 64KB (Max TCP packet size) seems too me, more then enough.

Reply:
If someone is doing batch changes we'll need more than 1MB in some cases, so it needs to be configurable either per route or per user in some cases.